src/status_im/ui/screens/accounts/recover/models.cljs

```diff
@@ -29,6 +47,27 @@
 
 ;;;; Handlers
 
+(defn set-phrase [recovery-phrase {:keys [db]}]
+  {:db (-> db
+           (assoc-in [:accounts/recover :passphrase] (string/lower-case recovery-phrase))
```

Comment:
i think in such cases better to use `update`
```clojure
(update :accounts/recover assoc :passphrase (string/lower-case recovery-phrase)
                                :passphrase-valid? (not (check-phrase-errors recovery-phrase)
```

Reply:
I don't really have a preference there, so sure why not.